Formula/a/aws-lc.rb

```diff
@@ -0,0 +1,28 @@
+class AwsLc < Formula
+  desc "General-purpose cryptographic library"
+  homepage "https://github.com/aws/aws-lc"
+  url "https://github.com/aws/aws-lc/archive/refs/tags/v1.51.2.tar.gz"
+  sha256 "7df65427f92a4c3cd3db6923e1d395014e41b1fcc38671806c1e342cb6fa02f6"
+  license all_of: ["Apache-2.0", "ISC", "OpenSSL", "MIT", "BSD-3-Clause"]
+
+  keg_only "it conflicts with OpenSSL"
+
+  depends_on "cmake" => :build
+  depends_on "go" => :build
+
+  uses_from_macos "perl"
+
+  def install
+    args = ["-DCMAKE_INSTALL_INCLUDEDIR=include", "-DCMAKE_INSTALL_BINDIR=bin"]
```

Comment:
I think this is a default value and doesn't need here, right?

Reply:
It's needed with how the project's `CMakeLists.txt` is set up. Relevant lines: https://github.com/aws/aws-lc/blob/v1.51.2/CMakeLists.txt#L137-L143 (`std_cmake_args` sets `CMAKE_INSTALL_LIBDIR` but not the others).

Reply:
```suggestion
    args = %w[
      -DCMAKE_INSTALL_INCLUDEDIR=include
      -DCMAKE_INSTALL_BINDIR=bin
    ]
```